weaviate/collections/classes/internal.py

```diff
@@ -352,15 +357,18 @@ def to_multi_target(
             `target_collection`
                 The collection in which the objects are stored. Can be either the name of the collection or the collection object itself.
         """
-        return _Reference[Properties, IReferences](
-            None,
+        return _Reference(
             target_collection.name
             if isinstance(target_collection, _CollectionBase)
             else target_collection,
             uuids,
         )
 
 
+WeaviateReference: TypeAlias = _Reference
+WeaviateReferences: TypeAlias = Mapping[str, WeaviateReference]
```

Comment:
I have the feeling we have to many classes that kind of sound the same, but have different usages and it is not obvious which one is used where (just thinking of all the references) - maybe we can make this more somehow obvious 🤔

Reply:
Will resolve in separate PR